pkg/pillar/types/zedroutertypes.go

```diff
@@ -1089,6 +1089,7 @@ type NetworkPortConfig struct {
 	WirelessCfg WirelessConfig
 	// TestResults - Errors from parsing plus success/failure from testing
 	TestResults
+	Cbattr map[string]string
```

Comment:
Makes sense to move this above the TestResults.

You need to update functions which compare and copy a NetworkPortConfig.

Reply:
The reason to move the field(s) above TestResults is that this *Config structure is actually a mixture of configuration and status where the status is the testresults.